std_execution.bs

```diff
@@ -2819,6 +3007,20 @@ namespace std::execution {
     using stop_token_of_t =
       remove_cvref_t&lt;decltype(get_stop_token(declval&lt;T>()))>;
 
+  // [execution.env], execution environments
+  namespace <i>exec-envs</i> { // exposition only
+    struct no_env;
+    struct <i>empty-env</i> {}; // exposition only
+    struct get_env_t;
+  }
+  using <i>exec-envs</i>::no_env;
+  using <i>exec-envs</i>::<i>empty-env</i>;
```

Comment:
Yeah. But it's an unspecified type, so it shouldn't really matter. Why does it give you pause?

Reply:
No wait. That's not right. `sender_of` requires `typed_sender`, and if *any* part of the sender traits potentially depends on the environment, it can't be a typed sender. So the `sender_of` check would fail.

Reply:
This is somewhat unsatisfying.